### PR TITLE
Wifip2p

### PR DESCRIPTION
**Pull request by RishabhRawatAndroid**
Add offline wifi p2p feature.

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -1,17 +1,17 @@
 import groovy.json.JsonSlurper
 
 buildscript {
-  repositories {
```
Comment (mhutti1) [inline app/build.gradle]: Please fix indentation here and in many other places. 2 space indentation, 4 space line continuation.

**Comment by abdulwd** [comment]
@mhutti1 Why is build not getting completed on this PR?

**Comment by RishabhRawatAndroid** [comment]
#616

**Comment by soloturn** [comment]
@RishabhRawatAndroid can you please rebase this onto current, and squash the commits? if you need hints how to do it dono't hesitate to ask ...

**Comment by prajurock** [comment]
@soloturn if @RishabhRawatAndroid is not working on this i will love to take this

**Comment by soloturn** [comment]
sure @prajurock , i checked @RishabhRawatAndroid s [repository]( https://github.com/RishabhRawatAndroid/kiwix-android/commits/wifip2p) and the [youtube  video]( https://www.youtube.com/watch?v=VlvmbrImH1I).

**Comment by prajurock** [comment]
Ok thanks.I really wanted to work on this issue.

**Comment by RishabhRawatAndroid** [comment]
@soloturn  @mhutti1  @prajurock First of all sorry for the late reply because I am preparing for my end term (final year) exam(which is very important for me). If you can give me 45 days then I can do this feature. On that time(when I was working on this feature) I was learning the wifi p2p but now I am pretty sure about the wifi p2p and I can do this. But this totally depends on you if you allow me then I would like to work on this feature.

**Comment by prajurock** [comment]
@soloturn What i will do now??

**Comment by kelson42** [comment]
@siddharth2010 @abdulwd @Aditya-Sood @RishabhRawatAndroid What should we do with this PR?

**Comment by Aditya-Sood** [comment]
@kelson42 #1276 resolves this. I also went through the code and it has not been completed yet, so I think we can close this.

**Pull request closed without merging**